lib/sablon/content.rb

```diff
@@ -196,6 +197,22 @@ def initialize(source, attributes = {})
         #
         super name, img_data
         @attributes = attributes
+
+        def width
+          if @attributes["properties"]
+            unconverted_width = @attributes["properties"][:width]
+            unit = @attributes["properties"][:unit]
```

Comment:
Instead of separating the size and unit it might be more user friendly to do something like `2cm` as the width/height value in the properties hash and then we can split the number and unit from the string for conversion into EMUs inside this method. That way it better mimics CSS which is something a lot of people are familar with.

Reply:
Good idea to make it CSS like! It definitely looks more user friendly now 👍 - done